implementations/rust/ockam/ockam_command/src/command.rs

```diff
@@ -58,6 +57,17 @@ impl OckamCommand {
         // This allows us to customize how we format the error messages and their content.
         let _hook_result = miette::set_hook(Box::new(|_| Box::new(ErrorReportHandler::new())));
 
+        let command_name = self.subcommand.name();
+
+        // Set the in-memory env var if needed
+        if let OckamSubcommand::Node(cmd) = &self.subcommand {
+            if let crate::node::NodeSubcommand::Create(c) = &cmd.subcommand {
+                if c.in_memory {
+                    std::env::set_var(OCKAM_SQLITE_IN_MEMORY, "true");
```

Comment:
Would it be hard to do it vice versa: Use the boolean flag as the source of truth and set it to true if the env variable is set? I think using environment as a global storage for variables isn't perfect

Reply:
Not in this case because it's just one variable, but I was following the way we configure the database [here](https://github.com/build-trust/ockam/blob/2ddf8ac6cacdaeecba0f794b242116555e676fb5/implementations/rust/ockam/ockam_api/src/cli_state/cli_state.rs#L255). Passing this as a variable would also mean adding a new field to `CliState` that would be exposing an implementation detail unnecessarily.

Reply:
It's also true that the default in-disk sqlite details are leaked into the CliState's `dir` field... and we are not consistent in defining the database options.

Reply:
IMO the right way to use environment variables, is to pull value from it and then populate it when it's needed. Otherwise, it's just a global variable (which is kinda a bad practice itself), but even slightly worse. It's okish to leave it that way to avoid extra work though. Up to you.

Reply:
Ok done! Let me know what you think 🙏